storage/index.md

```diff
@@ -100,6 +101,10 @@ mounts is to think about where the data lives on the Docker host.
   information. For instance, internally, swarm services use `tmpfs` mounts to
   mount [secrets](/engine/swarm/secrets.md) into a service's containers.
 
+- **[names pipes](https://docs.microsoft.com/en-us/windows/desktop/ipc/named-pipes)**: A `npipe`
```

Comment:
typo; `names pipes` -> `named pipes`

Also, (small nit): can you change "A npipe" to "An npipe"  ?